astro.config.mjs

```diff
@@ -37,32 +37,75 @@ export default defineConfig({
 			credits: false,
 			sidebar: [
 				{
-					label: "leadingNavLinks",
+					label: 'leadingNavLinks',
 					items: [
-						{ label: "Overview", link: "/overview" },
-						{ label: "Quickstart", link: "/" + latest_dir_name + "/tutorials/quickstart" }
+						{ label: 'Quickstart', link: '/' + latest_dir_name + '/tutorials/quickstart' },
+						{ label: 'Web Demo', link: '/' + latest_dir_name + '/wasmdemo' },
 					]
 				},
 				{
 					label: 'Start Here',
 					items: [
-						'overview'
+						'principles'
```

Comment:
The Markdown file was probably named "principles.md" and titled "Principles" originally, but I thought that was a less appealing title name than "Overview".

If you want to make the file change to "Principles", then that doesn't remove the need for having a general overview that all people (engineers and non-engineers alike) can go to and get a high level understanding before choosing if & whether to go deeper. Thus, you should create an Overview page.

Also, if you rename the file, make sure that the MD conversion tool in the `icu4x-docs` is adjusted accordingly.

Reply:
That's funny, I didn't know this was `principles.md` in Rust, it just very much reads like a "Principles" page, so I renamed it.

> that doesn't remove the need for having a general overview that all people (engineers and non-engineers alike) can go to and get a high level understanding before choosing if & whether to go deeper. Thus, you should create an Overview page.

Ok, but it also doesn't make the principles page a useful overview page. I acknowledge the need for an overview page, but this isn't it.